### ISSUE TITLE
[Bug]: Cannot run on MacOS

### ISSUE THREAD

**Issue by roopeux** [Sigil-Ebook/Sigil#672]
### Bug Description

MacOS: 11.6.4
Sigil.app-1.9.1-Mac.txz

I installed the newest Sigil on 10 March, but could not run it because verification failed.  Then in the Downloads directory, I got this response
```
$ codesign -dvvv Sigil.app
Sigil.app: code object is not signed at all
```

### Platform (OS)

macOS

### OS Version / Specifics

11.6.4

### What version of Sigil are you using?

1.9.1

**Comment by kevinhendricks** [comment]
The minimum deployment target for this build due to Qt is 10.12 and has been for the last couple of builds.

Also it is properly signed.

kbhend@KevinsiMac Sigil-1.9.1-Release % codesign -vv ./Sigil.app
./Sigil.app: valid on disk
./Sigil.app: satisfies its Designated Requirement

Sigil will build on macOS 10.11 with an earlier Qt version than Qt 5.12 but given the number of bug fixes in Qt and the CVE bugs fixes in QtWebEngine updates, I will will not be releasing builds with earlier versions of Qt.

FWIW Sigil wil eventually be moving to Qt 6.2.3 or later for the same reasons and Qt's minimum deployment target and therefore Sigil's will become 10.14 at that point.

Closing this as not a bug.

**Comment by kevinhendricks** [comment]
That said that release build is signed.  And I run it on my Big Sur laptop all of the time as well.
So I am not sure why you are getting that message unless you are confusing the "notarization" (like apps from the App Store use) and the error message Apple creates for un-notarized software -  even though that software is signed by official Apple developer certificates bought and paid for annually literally for years and years.

**Comment by kevinhendricks** [comment]
If so, Sigil is not "notarized" and will never be "notarized" as OpenSource software that does not use the Mac AppStore should never allow Apple to control things by its "approval".

That does not mean you can not run Sigil.  You just have to tell Apple you truly want to run Sigil (otherwise it will play "nurse maid" and try to prevent you from controlling your own Desktop machine much like they do with their walled-garden iPad, iPhone, iWatch, iTV, etc.

See issue #588 for instructions on how to tell Mac OS that yes you do truly want to run Sigil.  Then it will leave you alone.

**Comment by kevinhendricks** [comment]
PS -  Users have reported some serious bugs with the new search capabilities in Sigil 1.9.0 and its follow-on Sigil-1.91.  So you might want to wait for Sigil-1.9.2 which will hopefully be released in the next week or two or drop back down to Sigil-1.8.0 until things get cleared up.

**Comment by kevinhendricks** [comment]
No Problems.  

FYI - Sigil-1.9.2 will be released in the next day or two.  The Sigil 1.9.2 build has critical bug fixes and is meant to replace Sigil-1.9.0 and Sigil-1.9.1 as search as replace had some serious issues in both of those releases plus there was a crashing bug fixed.

You may want to wait a day for Sigil-1.9.2 or drop back to Sigil-1.8.0 until you are ready to upgrade.  I would not keep using Sigil-1.9.0 or Sigil-1.9.1 as it might eventually come back to bite you in the ass (so to speak).
Once Sigil-1.9.2 has been officially released we will be removing all of the Sigil-1.9.0 and Sigil-1.9.1 binaries to prevent issues (and future bug reports).
